packages/block-library/src/gallery/gallery-image.js

```diff
@@ -281,7 +282,10 @@ class GalleryImage extends Component {
 						value={ caption }
 						isSelected={ this.state.captionSelected }
 						onChange={ ( newCaption ) =>
-							setAttributes( { caption: newCaption } )
+							setAttributes( {
+								caption: newCaption,
+								alt: __unstableStripHTML( newCaption ),
```

Comment:
We don't want to necessarily override the `alt` which is a different attribute. I.e. a user can provide an `alt` and a `caption` (or no `caption`), in which case we'll use the alt and won't even look at the `caption`.

So we should strip the HTML from `caption` at save time, here: https://github.com/WordPress/gutenberg/pull/26082/files#diff-4283002b00fa661721fcb36f9eb609725082e259139407334ab318dd46746167R46

Reply:
Updated in https://github.com/WordPress/gutenberg/pull/26574/commits/898b5f89cf2e81d89d815b3ac59d98e575f01300.